src/dswx_sar/save_mgrs_tiles.py

```diff
@@ -236,135 +433,78 @@ def run(cfg):
             landcover_mask=landcover_mask,
             bright_water_fill=bright_water_mask,
             dark_land_mask=dark_land_mask,
-            layover_shadow_mask=layover_shadow_mask>0,
+            layover_shadow_mask=layover_shadow_mask > 0,
+            hand_mask=hand_mask,
+            inundated_vegetation=inundated_vegetation == 2,
             no_data=no_data_raster,
             )
 
-        dswx_sar_util.save_dswx_product(water_map==1,
+        dswx_sar_util.save_dswx_product(
```

Comment:
Are you running this twice?

Reply:
I changed the descriptions. one of them is BWTR and the other is WTR.